perl/p5-compress-raw-lzma/Portfile

```diff
@@ -25,4 +25,6 @@ long_description    Compress::Raw::Lzma provides an interface to the \
 if {${perl5.major} != ""} {
     depends_lib-append \
                     port:xz
+
```

Comment:
I could be wrong, but aren't tests already enabled by the perl5 portgroup?
https://github.com/macports/macports-ports/blob/3b72d44367b36051d162bb6f014ea6069058b72d/_resources/port1.0/group/perl5-1.0.tcl#L214

Reply:
Ah, you're right. Let me fix. Thank you!